base/cvd/cuttlefish/host/commands/cvd/selector/instance_database.cpp

```diff
@@ -38,7 +41,7 @@ constexpr const char kJsonGroups[] = "Groups";
 constexpr const unsigned UNSET_ID = 0;
 
 Result<std::string> GenUniqueGroupName(const cvd::PersistentData& data) {
-  auto name_prefix = GenDefaultGroupName() + "_";
+  auto name_prefix = std::string(kInternalGroupName) + "_";
```

Comment:
This comment is related to line 50, but I can't comment there ([because GitHub](https://github.com/isaacs/github/issues/1655) doesn't support comments on [expanded unmodified lines](https://github.com/orgs/community/discussions/4452)).

```c++
auto name = fmt::format("{}_{}", kInternalGroupName, i);
```

Reply:
Done